identity/src/main/java/com/android/identity/util/Logger.kt

```diff
@@ -242,4 +255,42 @@ object Logger {
             level: Int, tag: String, msg: String, throwable: Throwable?
         )
     }
-}
\ No newline at end of file
+
+    data class ActivityLogEntry(
+        val timestamp: String,
+        val transactionType: String,
+        val dataShared: String?,
+        val verifierIdentity: String?
+    )
```

Comment:
I think we probably want ActivityLogEntry to be a sealed class and subclasses for the various types we want to log ... DocumentPresentation, DocumentProvisioning, DocumentDeletion, DocumentUpdateCheck, DocumentNotification which I can think of in the top of my head. I would recommend starting with just implementing support for DocumentPresentation and DocumentUpdateCheck.